gulp/tasks/download.js

```diff
@@ -0,0 +1,34 @@
+var gulp = require('gulp');
+var fs = require('fs');
+var path = require('path');
+var request = require('request');
+
+var manifest = {
+  "content/en/governance.md": "https://raw.githubusercontent.com/iojs/io.js/v1.x/GOVERNANCE.md",
+  "content/en/working-groups.md": "https://raw.githubusercontent.com/iojs/io.js/v1.x/WORKING_GROUPS.md",
```

Comment:
a) it looks nicer to have them integrated in the website
b) the i18n teams have a consistent place to put their versions